PIL/PngImagePlugin.py

```diff
@@ -687,6 +687,9 @@ def _save(im, fp, filename, chunk=putchunk, check=0):
             mode = "%s;%d" % (mode, bits)
 
     # encoder options
+    if (im.info.get('showing', False)):
+        im.encoderinfo['compress_level'] = 1
+        im.info.pop('showing', None)
```

Comment:
`file` is a python builtin. It's bad form to have a local variable with that name. path, or temp_file are better.

But, looking now, it's in the function signature, which means finding any other place where it might have been called with kwargs.  At least it's an internal function so we can change it.